ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,407 @@
+              Your name
+              <TextInput
+                  data-test-id='user-full-name'
+                  style={styles.textInput}
+                  disabled={true}
+                  value={givenName + ' ' + familyName}/>
+            </FlexColumn>
+            <FlexColumn style={styles.textNormal}>
+              Your phone number
+              <TextInput
+                  data-test-id='user-phone-number'
+                  style={styles.textInput}
+                  onChange={(v) => validatePhoneNumber(v)}/>
+              {invalidPhoneNumberInput && <div data-test-id='invalidPhoneNumber' style={{color: colors.danger}}>
+                Invalid phone number input
+              </div>}
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{marginTop: '20px'}}>
+            <FlexColumn style={styles.textNormal}>
+              Your contact email address
+              <TextInput
+                  data-test-id='user-contact-email'
+                  disabled={true}
+                  style={styles.textInput}
+                  value={contactEmail}/>
+            </FlexColumn>
+            <FlexColumn style={styles.textNormal}>
+              Your researchallofus.org ID
+              <TextInput
+                  data-test-id='user-workbench-id'
+                  style={styles.textInput}
+                  disabled={true}
+                  value={username}/>
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{marginTop: '20px'}}>
+            <FlexColumn style={styles.textNormal}>
+              Your institution
+              <TextInput
+                  data-test-id='user-institution'
+                  style={styles.textInput}
+                  disabled={true}
+                  value={verifiedInstitutionalAffiliation.institutionDisplayName}/>
+            </FlexColumn>
+          </FlexRow>
+          <FlexRow style={{marginTop: '100px', justifyContent: 'flex-end'}}>
+            <Button type='secondary'
+                    style={{fontWeight: 400, padding: '0 18px', height: '40px', marginRight: '10px'}}
+                    onClick={() => onClose()}>
+              Cancel
+            </Button>
+            <Button data-test-id='next-button'
+                    type='primary'
+                    style={{fontWeight: 400, padding: '0 18px', height: '40px', width: '93px'}}
+                    disabled={!phoneNumber}
+                    onClick={() => {setCurrentStep(2); }}>
+              Next
+            </Button>
+          </FlexRow>
+        </FlexColumn>)],
+      [
+        2, () => (
+        <FlexColumn style={{justifyContent: 'space-evenly', width: '100%'}}>
+          <div style={styles.textHeader}>What payment method would you like to use?</div>
+          <FlexColumn style={{marginTop: '20px', width: '100%'}}>
+            <FlexRow style={{boxSizing: 'border-box', borderRadius: '8px', border: '1px solid #CCCFD4', marginBottom: '7px'}}>
```

Comment:
Pulled this locally and tried this out. What's happening is that the `RadioButton` is an element inside this `FlexRow`, and it's being resized because the information elements have different widths in these cases. Two solutions on the `RadioButton` style:

probably preferred:
```
flexShrink: 0
```

```
maxWidth: '17px'
```

I would suggest applying a similar fix for the checkbox. The label content is not quite aligned currently across these 3 sections - would be ideal if they were all aligned on a grid.